src/js/app.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready(function() {
+
+  $('#searchBtn').on('click', function(event) {
+    event.preventDefault();
+
+     $.get('https://www.reddit.com/search.json', {
+      q: valInput()
+    }).done(function(taco) {
```

Comment:
> There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.

Guilty of the second one. Although we use `taco` as a variable during lectures, don't actually use it in production code.

Also, `intoArray` is a little vague. Alternatives could include `processImages`, or `getImageUrls`